<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1224</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 1px solid white;
border-radius: 10px;
background: white;

background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 rgba(72, 145, 217, 255), stop: 1 rgba(255, 255, 255, 255));
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1201</width>
      <height>651</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>CRUD</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>521</width>
        <height>351</height>
       </rect>
      </property>
      <property name="title">
       <string>Employe</string>
      </property>
      <widget class="QLabel" name="label_49">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>CIN</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_50">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_51">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Prenom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_52">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Num Telephone</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_53">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>280</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Conge</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_54">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>100</y>
         <width>47</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Role</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_55">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>50</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Diplome</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_56">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Date de naissance</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_cin_4">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nom_4">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>90</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_prenom_4">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>130</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_tel_4">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>180</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_daten_4">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>240</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_diplome_4">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>40</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>License</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Master</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Doctorat</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ingenieurie</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBox_role_4">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>100</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Management</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Informatique</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Contabilite</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Reception</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="lineEdit_conge_4">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>280</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_57">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>150</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Duree de travail</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_duree_4">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>150</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>3 Mois</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6 Mois</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1 Ans</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3 Ans</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>+5 Ans</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pushButton_ajouter_4">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>280</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_modifier_4">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>280</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
     </widget>
     <widget class="QTableView" name="tableView_4">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>10</y>
        <width>541</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Rechercher_4">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>440</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_supprimer_4">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>440</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_p_4">
      <property name="geometry">
       <rect>
        <x>181</x>
        <y>400</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>CIN</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>TELEPHONE</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_Trier_4">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>470</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Trier</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_trie_4">
      <property name="geometry">
       <rect>
        <x>1030</x>
        <y>430</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>NOM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CONGE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CIN</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>380</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_58">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>390</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_cins_4">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>400</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_r_4">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>380</y>
        <width>491</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="stat">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>500</y>
        <width>75</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Stat</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>QUIZ</string>
     </attribute>
     <widget class="QLabel" name="label_59">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>20</y>
        <width>121</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 32pt &quot;MS UI Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>QUIZ</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_60">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>150</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Question 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_61">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>270</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Question 5</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_62">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>370</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Question 3</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_63">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>150</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Question 4</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_64">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>260</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Question 2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_valider_4">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>490</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="Q1R1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Réponse 1</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="Q1R2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>180</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Réponse 2</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="Q1R3">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>180</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Réponse 3</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="Q5R1">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>310</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Réponse 1</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="Q4R1">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>180</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Réponse 1</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="Q3R1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Réponse 1</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="Q2R1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Réponse 1</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="Q4R2">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>180</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Réponse 2</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="Q3R2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>400</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Réponse 2</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_10">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>310</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Réponse 2</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="Q2R2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>290</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Réponse 2</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_12">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>310</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Réponse 3</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="Q4R3">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>180</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Réponse 3</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="Q3R3">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>400</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Réponse 3</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="Q2R3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>290</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Réponse 3</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1224</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
